### PR TITLE
Feature/mic 2555 clinical trial efficacy

### PR DESCRIPTION
**Pull request by mattkappel**
This implements the MIC-2555, including scenarios for using either the population-based or randomized clinical trial-based values. These will show up in results as `hazard_rate_source = ['population', 'clinical_trial']`.

Tested with a full parallel run with 2640 permutations of scenarios with results that look reasonable on inspection. Note: As in previous models, deaths from other_causes are represented by deaths in the susceptible state.

```
file = Path(f"{base_dir}/count_data/deaths.csv")
deaths = pd.read_csv(file)
deaths.groupby(["measure", "cause", "scenario", "hazard_rate_source"]).value.sum()
[7]:
measure  cause                            scenario     hazard_rate_source
death    multiple_myeloma_1               alternative  clinical_trial           23102.0
                                                       population               23223.0
                                          baseline     clinical_trial           23085.0
                                                       population               23211.0
         multiple_myeloma_2               alternative  clinical_trial           17174.0
                                                       population               17632.0
                                          baseline     clinical_trial           17237.0
                                                       population               17643.0
         multiple_myeloma_3               alternative  clinical_trial           13930.0
                                                       population               13617.0
                                          baseline     clinical_trial           13956.0
                                                       population               13586.0
         multiple_myeloma_4               alternative  clinical_trial           10039.0
                                                       population                9690.0
                                          baseline     clinical_trial           10059.0
                                                       population                9741.0
         multiple_myeloma_5               alternative  clinical_trial           14027.0
                                                       population               14577.0
                                          baseline     clinical_trial           14032.0
                                                       population               14583.0
         other_causes                     alternative  clinical_trial               0.0
                                                       population                   0.0
                                          baseline     clinical_trial               0.0
                                                       population                   0.0
         susceptible_to_multiple_myeloma  alternative  clinical_trial        10919645.0
                                                       population            10919645.0
                                          baseline     clinical_trial        10919645.0
                                                       population            10919645.0
```

### CODE REVIEW

**Review by KjellSwedin — APPROVED** [review]

File: src/vivarium_csu_sanofi_multiple_myeloma/constants/data_values.py [hunk 1]
```diff
@@ -57,6 +58,45 @@
     (models.MULTIPLE_MYELOMA_5_STATE_NAME, models.TREATMENTS.residual, True): (0.952, 0.744, 1.145),
 }
 
+# Randomized Clinical Trial Progression Free Survival Hazard Ratios
+RCT_PFS_HR = {
```
Comment (aflaxman) [inline src/vivarium_csu_sanofi_multiple_myeloma/constants/data_values.py:62]: @alibow can you double that these all got copied in correctly?

File: src/vivarium_csu_sanofi_multiple_myeloma/constants/data_values.py [hunk 2]
```diff
@@ -57,6 +58,45 @@
...
+
+# Fill in 3rd and 4th lines using the same values as in the 2nd line
+LATER_LINES = [models.MULTIPLE_MYELOMA_3_STATE_NAME, models.MULTIPLE_MYELOMA_4_STATE_NAME]
+ACTUAL_TREATMENTS = [t for t in models.TREATMENTS if t != models.TREATMENTS.not_treated]
+for line, treatment, retreated in itertools.product(LATER_LINES, ACTUAL_TREATMENTS, [True, False]):
+    RCT_PFS_HR[(line, treatment, retreated)] = RCT_PFS_HR[(models.MULTIPLE_MYELOMA_2_STATE_NAME, treatment, retreated)]
+
+# Randomized Clinical Trial Overall Survival Hazard Ratios
+RCT_OS_HR = {
+    (models.MULTIPLE_MYELOMA_1_STATE_NAME, models.TREATMENTS.isatuximab, False): (0.760, 0.645, 0.895),
```
Comment (aflaxman) [inline src/vivarium_csu_sanofi_multiple_myeloma/constants/data_values.py:83]: @alibow can you double that these all got copied in correctly?

Comment (alibow) [reply src/vivarium_csu_sanofi_multiple_myeloma/constants/data_values.py:83]: Looks good to me!

File: src/vivarium_csu_sanofi_multiple_myeloma/constants/data_values.py (hunk 1 above)
Comment (alibow) [reply src/vivarium_csu_sanofi_multiple_myeloma/constants/data_values.py:62]: Looks good to me!

**Pull request merged**